docs/scripting/callbacks/OnClientMessage.md

```diff
@@ -29,7 +29,7 @@ This callback does not handle returns.
 ```c
 public OnClientMessage(color, text[])
 {
-    if(strfind(text,"Bank Balance: $0") != -1) SendChat("I am poor :(");
+    if (strfind(text,"Bank Balance: $0") != -1) SendChat("I am poor :(");
```

Comment:
`SendChat` is not a SA-MP or open.mp API. Also this should be in a compound block as the guide recommends always using braces even for blocks of only a single statement.